examples/statemachine/package.json

```diff
@@ -60,7 +60,9 @@
         "chalk": "~5.3.0",
         "commander": "~11.0.0",
         "langium": "~2.0.0",
-        "lodash": "^4.17.21"
+        "lodash": "^4.17.21",
+        "vscode-languageclient": "~9.0.1",
```

Comment:
These deps were not needed before this change. Why now?

Reply:
That seems to have been missing all the time.